scripts/build-installer-win64.bat

```diff
@@ -0,0 +1,102 @@
+rem DEPENDENCIES:
+rem   1. Node.js ('npm' binary in PATH)
+rem   2. 7zip    ('7z'  binary in PATH)
+
+set DEFAULT_CARDANO_BRANCH=cardano-sl-0.4
+set MIN_CARDANO_BYTES=50000000
+set LIBRESSL_VERSION=2.5.3
+
+set DAEDALUS_VERSION=%1
+@if [%DAEDALUS_VERSION%]==[] (@echo FATAL: DAEDALUS_VERSION [argument #1] was not provided
+    exit /b 1);
+set CARDANO_BRANCH=%2
+@if [%CARDANO_BRANCH%]==[]   (@echo NOTE: CARDANO_BRANCH [argument #2] was not provided, defaulting to %DEFAULT_CARDANO_BRANCH%
+    set CARDANO_BRANCH=%DEFAULT_CARDANO_BRANCH%);
+
+@echo Building Daedalus version: %DAEDALUS_VERSION%
+@echo ..with Cardano branch:     %CARDANO_BRANCH%
+
+call npm install
+@if %errorlevel% neq 0 (@echo FAILED: npm install
+    exit /b 1)
+
+@echo Obtaining Cardano from branch %CARDANO_BRANCH%
+rmdir /s/q node_modules\daedalus-client-api 2>nul
+mkdir      node_modules\daedalus-client-api
+
+pushd node_modules\daedalus-client-api
+    powershell -Command "try { $wc = New-Object net.webclient; $wc.Downloadfile('https://ci.appveyor.com/api/projects/jagajaga/cardano-sl/artifacts/CardanoSL.zip?branch=%CARDANO_BRANCH%', 'CardanoSL.zip'); } catch { exit 1; }"
+    @if %errorlevel% neq 0 (@echo FAILED: powershell -Command "try { $wc = New-Object net.webclient; $wc.Downloadfile('https://ci.appveyor.com/api/projects/jagajaga/cardano-sl/artifacts/CardanoSL.zip?branch=%CARDANO_BRANCH%', 'CardanoSL.zip'); } catch { exit 1; }"
+	popd & exit /b 1)
+    @for /F "usebackq" %%A in ('CardanoSL.zip') do set size=%%~zA
+    if %size% lss %MIN_CARDANO_BYTES% (@echo FAILED: CardanoSL.zip is too small: threshold=%MIN_CARDANO_BYTES%, actual=%size% bytes
+        popd & exit /b 1)
+
+    7z x CardanoSL.zip -y
+    @if %errorlevel% neq 0 (@echo FAILED: 7z x CardanoSL.zip -y
+	popd & exit /b 1)
+    del CardanoSL.zip
+popd
+
+move   node_modules\daedalus-client-api\log-config-prod.yaml installers\log-config-prod.yaml
+move   node_modules\daedalus-client-api\cardano-node.exe     installers\
+move   node_modules\daedalus-client-api\cardano-launcher.exe installers\
+del /f node_modules\daedalus-client-api\*.exe
+
+rem @echo Obtaining LibreSSL %LIBRESSL_VERSION%
+rem https://ftp.openbsd.org/pub/OpenBSD/LibreSSL/libressl-%LIBRESSL_VERSION%-windows.zip
```

Comment:
Is the plan to eventually install libressl automatically, perhaps conditional upon a var? If so, consider adding a comment indicating the plan.

The thing about commented code w/o an explanation is people don't know whether it indicates past failure or unfulfilled future plans.